uint8 damp
---
bool success
